app/services/github_service.rb

```diff
@@ -28,6 +28,12 @@ def handle_event
     end
   end
 
+  def resolve_event_name(event)
```

Comment:
Can you add a brief comment explaining why this is necessary?

Reply:
Yep! Sorry, as a hotfix I mean to do it quick. We are receiving `pull_request_review` instead of `review` and `pull_request_review_comment` instead of `review_comment`. So in this way we make sure to get the appropriate event name